src/auth/passwordreset/tests/ResetComponent.test.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ResetForm from '../components/ResetForm';
+import ResetComponent from '../components/ResetComponent';
+
+jest.mock('../containers/withPasswordState', () => C => p => <C {...p} />)
+jest.mock('../components/ResetForm', () => () => <div />)
+
+const mockProps = {
+  request_success: true,
+  resetPasswordAction: jest.fn(),
+  request_errors: {},
+  isLoading: true,
+}
+
+const EMPTY_EMAIL_ERROR = 'Email field required';
```

Comment:
Great job on generally using constants, you can actually put these in some file and export them such that you can use them the `ResetForm.jsx` as well...

Reply:
This has been resolved, thank you.